terraform/modules/ops/main.tf

```diff
@@ -21,25 +23,9 @@ resource "google_project_iam_member" "pubsub_publisher_iam_member" {
   ]
 }
 
-# Container Hosting
-
-# Artifact Registry API enablement is eventually consistent
```

Comment:
Is there a reason we're deleting these comments?

(Are they no longer relevant? If they are, I'd argue this information should be documented _somewhere_, e.g. an `.md` file.)

Reply:
These comments were moved to services.tf.